### ISSUE TITLE
NVDA Repeats Controls Twice On Activity View Switcher Control

### ISSUE THREAD

**Issue by jhomme** [microsoft/vscode#103247]
Issue Type: <b>Bug</b>

Switch to this control and navigate through it. We can eliminate the title attribute if that is what is duplicating the information.

VS Code version: Code - Insiders 1.48.0-insider (731f9c25632dbbf01ee3a7892ad9d2791fe0260c, 2020-07-24T05:57:10.466Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz (4 x 2397)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.87GB (3.37GB free)|
|Process Argv|--crash-reporter-id ab8e342c-4a51-45aa-9ade-8ae02d47f2e7|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
author-stats|ada|1.0.4
vscode-languagetool|ada|3.8.0
Bookmarks|ale|11.3.1
spellright|ban|3.0.50
json-tree-view|Cha|2.6.0
vscode-markdownlint|Dav|0.36.1
vscode-pandoc|Dou|0.0.8
vscode-hugo-snippets|fiv|0.4.1
readabilitycheck|jem|1.3.0
markdown-shortcuts|mdi|0.12.0
python|ms-|2020.7.96456
vscode-hugo|rus|0.24.0
rewrap|stk|1.13.0
markdown-all-in-one|yzh|3.1.0

</details>

**Comment by isidorn** [comment]
@jhomme root cause is https://github.com/nvaccess/nvda/issues/7841
Can you please comment on that issue?
We have to set the title attribute for users reallying on the mouse hover information to discover keybindings. All other screen readers handle this correctly, so I believe this should be fixed in NVDA.

**Comment by jhomme** [comment]
Hi,

I commented on the issue, but I still feel that the ARIA label should be removed if it duplicates the on-screen label. I also still feel that the title attribute should be removed if it is the same as either the on-screen label or the ARIA label. If the title attribute is different, it should not be removed. There is never a need to make the ARIA label duplicate an on-screen label, especially if the on-screen label is inside a label tag and coded correctly. 

 

==========

Jim Homme

Skype: jim.homme

FreeChess: jhomme

Twitter: jimhomme <https://twitter.com/jimhome> 

Facebook: jimhomme

Website: jimhomme.com <https://www.jimhomme.com/>

**Comment by isidorn** [comment]
Same root cause as https://github.com/microsoft/vscode/issues/96077
I prefer to only have one issue around for that, since the bottom issue is on the nvda side, thus closing this.
